libs/vault/src/cipher-form/components/login-details-section/login-details-section.component.ts

```diff
@@ -65,10 +65,15 @@ export class LoginDetailsSectionComponent implements OnInit {
   newPasswordGenerated: boolean;
 
   /**
-   * Whether the TOTP field can be captured from the current tab. Only available in the browser extension.
+   * Whether the TOTP field can be captured from the current tab. Only available in the browser extension and
+   * when not in a popout window.
    */
   get canCaptureTotp() {
-    return this.totpCaptureService != null && this.loginDetailsForm.controls.totp.enabled;
+    return (
+      window.location.href.indexOf("uilocation=popout") === -1 &&
```

Comment:
🎨 The `BrowserPopupUtils` class has this logic wrapped into a helper method, that can be used here for future proofing if the logic were to ever change.

```suggestion
      BrowserPopupUtils.inPopout(window) &&
```

Reply:
Hey @nick-livefront! I was under the impression that importing app-specific code from was restricted. If there's a way around this please let me know. ;)

Reply:
Hmm I was unaware of that and I can't find another usage of `BrowserPopupUtils` within `libs/*`. 

@shane-melton @Jingo88 @gbubemismith Can you confirm Jordan's rule? Is there another way to consume `BrowserPopupUtils` or just accomplish knowing if the window is popped out?

Reply:
He's correct, we can't import the browser specific code directly in the `/libs/vault` library as it's used by the other clients and will fail to build those clients.

For that reason, we did already create a `TotpCaptureService` abstraction related to this and it has a browser specific implementation (`BrowserTotpCaptureService`). We could add a new method to the to that abstraction, like `canCaptureTotp()` that uses the `BrowserPopupUtils` method in the implementation. (which is allowed because the implementation is only used/registered in the Browser client)

Reply:
@jaasen-livefront Let me know what you think and/or if you have any questions on that approach!

Reply:
@shane-melton Will do thanks!

Reply:
Thanks @shane-melton, I guess I haven't run into that scenario yet 😄